ci/osp17_functional_tests.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: "Run functional test playbooks"
+  hosts:  "{{ cifmw_target_hook_host | default('localhost') }}"
+  tasks:
+    - name: Search for stackrc file
+      find:
+        paths: "/"
+        patterns: "stackrc"
+      register: stackrc_files
+
+    - debug:
+        msg: "Found stackrc file at {{ item.path }}"
+      loop: "{{ stackrc_files.files }}"
+
+    - name: Test service API endpoint(metric) for autoscaling
+      ansible.builtin.shell: |
+        source ~/stackrc
```

Comment:
Probably it could not find stackrc in this location. Maybe you need to get the patch from stackrc_files variable ?